spec/schemas/_core.reindex_rethrottle.yaml

```diff
@@ -64,10 +64,10 @@ components:
           description: The number of documents that were successfully deleted.
           type: number
         noops:
-          description: The number of documents that were ignored because the script used for the reindex returned a `noop` value for `ctx.op`.
+          description: The number of documents that were ignored because the script used for the reindex and returned a `noop` value for `ctx.op`.
```

Comment:
```suggestion
          description: The number of documents that were ignored because the script used for the reindex operation returned a `noop` value for `ctx.op`.
```